src/request.rs

```diff
@@ -206,9 +206,74 @@ pub enum Request {
+            GetMDataVersion(_) => Response::GetMDataVersion(Err(error)),
+            ListMDataEntries(_) => Response::ListMDataEntries(Err(error)),
+            ListMDataKeys(_) => Response::ListMDataKeys(Err(error)),
+            ListMDataValues(_) => Response::ListMDataValues(Err(error)),
+            SetMDataUserPermissions { .. } => Response::Mutation(Err(error)),
+            DelMDataUserPermissions { .. } => Response::Mutation(Err(error)),
+            ListMDataPermissions(_) => Response::ListMDataPermissions(Err(error)),
+            ListMDataUserPermissions { .. } => Response::ListMDataUserPermissions(Err(error)),
+            MutateSeqMDataEntries { .. } => Response::Mutation(Err(error)),
+            MutateUnseqMDataEntries { .. } => Response::Mutation(Err(error)),
+            // AData
+            PutAData(_) => Response::Mutation(Err(error)),
+            GetAData(_) => Response::GetAData(Err(error)),
+            GetADataShell { .. } => Response::GetADataShell(Err(error)),
+            GetADataValue { .. } => Response::GetADataValue(Err(error)),
+            DeleteAData(_) => Response::Mutation(Err(error)),
+            GetADataRange { .. } => Response::GetADataRange(Err(error)),
+            GetADataIndices(_) => Response::GetADataIndices(Err(error)),
+            GetADataLastEntry(_) => Response::GetADataLastEntry(Err(error)),
+            GetADataPermissions { .. } => Response::GetADataPermissions(Err(error)),
+            GetPubADataUserPermissions { .. } => Response::GetPubADataUserPermissions(Err(error)),
+            GetUnpubADataUserPermissions { .. } => {
+                Response::GetUnpubADataUserPermissions(Err(error))
+            }
+            GetADataOwners { .. } => Response::GetADataOwners(Err(error)),
+            AddPubADataPermissions { .. } => Response::Mutation(Err(error)),
+            AddUnpubADataPermissions { .. } => Response::Mutation(Err(error)),
+            SetADataOwner { .. } => Response::Mutation(Err(error)),
+            AppendSeq { .. } => Response::Mutation(Err(error)),
+            AppendUnseq(_) => Response::Mutation(Err(error)),
+            // Coins
+            TransferCoins { .. } => Response::Transaction(Err(error)),
+            GetBalance => Response::GetBalance(Err(error)),
+            CreateBalance { .. } => Response::Transaction(Err(error)),
+            // Login Packet
+            CreateLoginPacket { .. } => Response::Mutation(Err(error)),
+            CreateLoginPacketFor { .. } => Response::Mutation(Err(error)),
+            UpdateLoginPacket { .. } => Response::Mutation(Err(error)),
+            GetLoginPacket(..) => Response::GetLoginPacket(Err(error)),
+            // Client (Owner) to SrcElders
+            ListAuthKeysAndVersion => Response::ListAuthKeysAndVersion(Err(error)),
+            InsAuthKey { .. } => Response::Mutation(Err(error)),
+            DelAuthKey { .. } => Response::Mutation(Err(error)),
+        }
```

Comment:
You can reduce the amount of code here a bit by grouping all the requests that map to `Response::Mutation`:

```rust
Request::PutIData(_)
    | Request::DeleteUnpubIData(_)
    ...
    | Request::DelAuthKey { .. } => Response::Mutation(Err(error)),
```